### PR TITLE
 Decrease pr_limit for Python 3.11 migration (with travis-ci)

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

cc @jakirkham, @mbargull 
Closes: gh-3589

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -2,3 +2,7 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
+noarch_platforms:
+  - linux_aarch64
```
Comment (mbargull) [inline conda-forge.yml:8]: Ah, good one! Makes sense.

**Review by mbargull — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

**Comment by isuruf** [comment]
It's 600 jobs in queue now. :smile:
Might need to pause the migration for a while.

**Comment by jakirkham** [comment]
Yeah Marcel saw the same thing

Arrow builds were starting. So have just been canceling those and providing an explanation. Maybe we can revisit after some other things have been built and the queue has calmed down

**Comment by mbargull** [comment]
Yep, let's pause or go with a limit of 5 or so.

**Comment by jakirkham** [comment]
We can try 5

**Comment by mbargull** [comment]
-> gh-3590

**Pull request merged**